airflow/utils/dag_processing.py

```diff
@@ -724,26 +530,27 @@ class DagFileProcessorManager(LoggingMixin):
     """
 
     def __init__(self,
-                 dag_directory,
-                 file_paths,
-                 max_runs,
-                 processor_factory,
-                 processor_timeout,
-                 signal_conn,
-                 async_mode=True):
-        self._file_paths = file_paths
-        self._file_path_queue = []
-        self._dag_directory = dag_directory
-        self._max_runs = max_runs
-        self._processor_factory = processor_factory
-        self._signal_conn = signal_conn
-        self._async_mode = async_mode
+                 dag_directory: str,
+                 file_paths: List[str],
+                 max_runs: int,
+                 processor_factory: Callable[[str, List[Any]], AbstractDagFileProcessor],
+                 processor_timeout: timedelta,
+                 signal_conn: Connection,
+                 async_mode: bool = True):
+        self._file_paths: List[str] = file_paths
+        self._file_path_queue: List[str] = []
+        self._dag_directory: str = dag_directory
+        self._max_runs: int = max_runs
+        self._processor_factory: Callable[[str, List[Any]], AbstractDagFileProcessor] = processor_factory
+        self._signal_conn: Connection = signal_conn
+        self._async_mode: bool = async_mode
```

Comment:
Style point that w should decide upon: In cases like this where th function arg is typed we don't also need to specify the type on the instance attribute - mypy will do that automatically.

So this could be written as:

```
    def __init__(self:
                 dag_directory: str,
                 file_paths: List[str],
                 max_runs: int,
                 processor_factory: Callable[[str, List[Any]], AbstractDagFileProcessor],
                 processor_timeout: timedelta,
                 signal_conn: Connection,
                 async_mode: bool = True):
        self._file_paths file_paths
        self._file_path_queue: List[str] = []
        self._dag_directory = dag_directory
        self._max_runs = max_runs
        self._processor_factory = processor_factory
        self._signal_conn = signal_conn
        self._async_mode = async_mode
```

Reply:
Yeah. I can fix it here. And I agree it's not needed.